mcs/build/profiles/mobile_static.make

```diff
@@ -44,8 +44,12 @@ NO_VTS_TEST = yes
 # Note need for trailing comma. If you add, keep it
 PROFILE_TEST_HARNESS_EXCLUDES = MobileNotWorking,
 
-ifndef MONO_DISABLE_GSHAREDVT
-GSHAREDVT_FLAG = -O=gsharedvt
+GSHAREDVT_FLAG := -O=gsharedvt
+
+GSHAREDVT_SUPPORTED = $(shell MONO_ENV_OPTIONS="-O=gsharedvt --aot=metadata-only,outfile=/dev/null" $(MCS) 2>&1 | grep -q "gsharedvt not supported" && echo 0 || echo 1)
```

Comment:
See https://github.com/akoeplinger/mono/commit/6ac50ede2a19a7502519fe421d1011093da24a3c#commitcomment-16412609.
I don't know of a good way to propagate the autoconf vars into the mcs/ build system.